site/en/r2/tutorials/eager/custom_training.ipynb

```diff
@@ -74,10 +74,10 @@
         "id": "k2o3TTG4TFpt"
       },
       "source": [
-        "In the previous tutorial we covered the TensorFlow APIs for automatic differentiation, a basic building block for machine learning.\n",
-        "In this tutorial we will use the TensorFlow primitives introduced in the prior tutorials to do some simple machine learning.\n",
+        "In the previous tutorial, you covered the TensorFlow APIs for automatic differentiation—a basic building block for machine learning.\n",
+        "In this tutorial, you will use the TensorFlow primitives introduced in the prior tutorials to do some simple machine learning.\n",
         "\n",
-        "TensorFlow also includes a higher-level neural networks API (`tf.keras`) which provides useful abstractions to reduce boilerplate. We strongly recommend those higher level APIs for people working with neural networks. However, in this short tutorial we cover neural network training from first principles to establish a strong foundation. "
+        "TensorFlow also includes `tf.keras`—a high-level neural network API, which provides useful abstractions to reduce boilerplate and makes TensorFlow easier to use without sacrificing flexibility and performance. We strongly recommend [this high-level API](https://www.tensorflow.org/beta/guide/keras/overview) for people working with neural networks. However, in this short tutorial you will learn how to train a neural network from first principles to establish a strong foundation."
```

Comment:
https://www.tensorflow.org/beta/guide/keras/overview

Prefer relative links so you can stay in tensorflow.org/github/colab. Use:

\[this high-level API](../../guide/keras/overview.ipynb)

Reply:
Got it @MarkDaoust, thanks. Noticed these relative links after the PR elsewhere and was going to ask 👍